README.md

```diff
@@ -307,3 +307,12 @@ make cluster-down
 # Run function test
 make functest
 ```
+
+In order to run functests on external cluster
+with CNAO installed:
```

Comment:
Add an inline example how to patch deployed CNAO to enable KMP if you really want to document this. User should be fully aware of it.